File: leetcode/1662.检查两个字符串数组是否相等.cpp
#include <iostream>
#include <vector>
using namespace std;
/*
 * @lc app=leetcode.cn id=1662 lang=cpp
 *
 * [1662] 检查两个字符串数组是否相等
 */

// @lc code=start
class Solution
{
public:
    bool arrayStringsAreEqual(vector<string> &word1, vector<string> &word2)
    {
        auto join = [](vector<string> &word) -> string
        {
            string res = "";
            for (auto &&w : word)
            {
                res += w;
            }
            return res;
        };
        return join(word1) == join(word2);
    }
};
// @lc code=end

File: leetcode/1662.检查两个字符串数组是否相等.py
#
# @lc app=leetcode.cn id=1662 lang=python3
#
# [1662] 检查两个字符串数组是否相等
#
from typing import List


# @lc code=start
class Solution:
    def arrayStringsAreEqual(self, word1: List[str], word2: List[str]) -> bool:
        return "".join(word1) == "".join(word2)
# @lc code=end
